WcaOnRails/lib/country_city_validators/au_city_validator.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+AU_STATES = %w(
+  Australian\ Capital\ Territory
+  New\ South\ Wales
+  Northern\ Territory
+  Queensland
+  South\ Australia
+  Tasmania
+  Victoria
+  Western\ Australia
+).to_set
+
+CityValidator.add_validator_for_country "AU", CityCommaRegionValidator.new(type_of_region: "state/territory", valid_regions: AU_STATES)
```

Comment:
would "state or territory" be nicer here? The error message our users are going to get will look something like "foo is not a valid state/territory" or ""foo is not a valid state or territory".